templates/car_input.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <h1>Welcome to the Automotive Solution Tool!</h1>
+</head>
+<body>
+
+    <p>Please fill out the boxes below to search for your desired car:</p>
+
+    <form method="GET" action="get_car">
+        
+        <form>
+            Make: <input type="text" name="make"><br />
```

Comment:
`<label>Make:</label>`  is an easy way to set our inputs with associated label text.

Change `<br />` to `<br>' and place is on a separate line

`<br>`
`<label>`
`<br>`
`<input>`

^ this will allow you to sort each input in blocks of code